opentelemetry-jaeger/src/lib.rs

```diff
@@ -299,6 +306,14 @@ impl PipelineBuilder {
         }
     }
 
+    /// Config whether to export information of instrumentation library.
+    pub fn export_instrumentation_library(self, export: bool) -> Self {
```

Comment:
I suggest we start the function with `with_` to follow the convention. Maybe something like `with_instrumentation_library_tags`.